backend/controllers/workoutController.js

```diff
@@ -0,0 +1,100 @@
+const Workout = require('../models/workoutModel')
```

Comment:
Right now the backend does not compile. Since the workout stuff is still implemented within our backend we need the workoutModel.js file for the backend to compile. I tested the code after adding the workoutModel.js file back and it seems to solve the issue.